src/index.ts

```
@@ -233,17 +235,53 @@ For more information, visit: https://www.globalping.io
+		if (this.state) {
+			return this.setState({
+				...this.state,
+				oAuth: state,
+			});
+		}
+		else {
+			return this.setState({
+				...this.initialState,
+				oAuth: state,
+			});
+		}
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)